app/src/ui/toolbar/branch-dropdown.tsx

```diff
@@ -168,12 +173,29 @@ export class BranchDropdown extends React.Component<IBranchDropdownProps> {
         showDisclosureArrow={canOpen}
         progressValue={progressValue}
         buttonClassName={buttonClassName}
+        onDragOver={this.onDragOver}
       >
         {this.renderPullRequestInfo()}
       </ToolbarDropdown>
     )
   }
 
+  /**
+   * Method to capture when something is dragged over the branch dropdown.
+   */
+  private onDragOver = (event: React.DragEvent<HTMLDivElement>): void => {
+    event.preventDefault()
+
+    // If the cherry picking state is initiated, open the branch menu.
```

Comment:
I'd be a bit more specific maybe:
```suggestion
    // If the cherry picking state is initiated, we assume the user is
    // dragging commits. Therefore, we should open the branch menu.
```

Reply:
Nice clarification.